topics/Routing_in_Ktor.md

```diff
@@ -122,6 +124,54 @@ To access a parameter value inside the route handler, use the `call.parameters`
 A path parameter with a tailcard (`{param...}`) matches all the rest of the URL path and puts multiple values for each path segment into parameters using `param` as a key. For example, `/user/{param...}` matches `/user/john/settings`.
 To access path segments' values inside the route handler, use `call.parameters.getAll("param")`. For the example above, the `getAll` function will return an array containing the _john_ and _settings_ values.
 
+### Regular expression {id="regular_expression"}
+
+Regular expressions can be used with all defining route handlers functions: `route`, `get`, `post`, and so on. For example:
+
+```kotlin
+import io.ktor.server.routing.*
+import io.ktor.server.response.*
+
+routing {
+    get(Regex(".+/hello")) {
+        call.respondText("Hello")
+    }
+}
+```
+
+#### Named groups
```

Comment:
I would name it using the purpose: `Accessing path parts in handler` and next explain how to use groups for that